### PR TITLE
Init

### PR DESCRIPTION
**Pull request by Jamie-BitFlight**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (osterman) [inline LICENSE:189]: Update

File: examples/marbot/main.tf [hunk 2]
```diff
@@ -0,0 +1,34 @@
+variable "marbot_endpoint_id" {
+  default = ""
```
Comment (osterman) [inline examples/marbot/main.tf:2]: Add description

File: examples/marbot/main.tf [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+variable "region" {
+  default = "eu-west-2"
```
Comment (osterman) [inline examples/marbot/main.tf:6]: Add description

File: outputs.tf [hunk 4]
```diff
@@ -0,0 +1,3 @@
+output "sns_topic_arn" {
+  value = "${local.sns_topic_arn}"
```
Comment (osterman) [inline outputs.tf:2]: Add description

File: main.tf [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+resource "aws_sns_topic_policy" "default" {
+  count  = "${var.add_sns_policy != "true" && var.sns_topic_arn != "" ? 0 : 1}"
+  arn    = "${local.sns_topic_arn}"
+  policy = "${data.aws_iam_policy_document.sns_topic_policy.json}"
+}
+
+data "aws_iam_policy_document" "sns_topic_policy" {
+  policy_id = "__default_policy_ID"
```
Comment (osterman) [inline main.tf:15]: I am not familiar with this convention

File: alarms.tf [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+    Resource           = "${var.function_resource}"
+  }
+}
+
+## Units: Count
+## Measures the number of Lambda function invocation attempts that were throttled 
+## due to invocation rates exceeding the customer’s concurrent limits (error code 429). 
+## Failed invocations may trigger a retry attempt that succeeds.
+resource "aws_cloudwatch_metric_alarm" "lambda_throttles" {
+  alarm_name          = "lambda_throttles"
```
Comment (osterman) [inline alarms.tf:51]: Use label

File: alarms.tf [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+## retry attempt that succeeds. This includes:
+## - Handled exceptions (for example, context.fail(error))
+## - Unhandled exceptions causing the code to exit
+## - Out of memory exceptions
+## - Timeouts
+## - Permissions errors
+## This does not include invocations that fail due to invocation rates exceeding default concurrent 
+## limits (error code 429) or failures due to internal service errors (error code 500).
+resource "aws_cloudwatch_metric_alarm" "lambda_errors" {
+  alarm_name          = "lambda_errors"
```
Comment (osterman) [inline alarms.tf:25]: Use label

File: README.md [hunk 8]
```diff
@@ -1,2 +1,19 @@
 # terraform-aws-lambda-cloudwatch-alarms
 Terraform module for creating a set of Lambda alarms and outputting to an endpoint
```
Comment (aknysh) [inline README.md:2]: Terraform module for creating a set of Lambda alarms and outputting to an SNS endpoint

File: examples/marbot/main.tf (hunk 2 above)
Comment (aknysh) [reply examples/marbot/main.tf:2]: Describe what is marbot.
Looks like it's this https://marbot.io/

File: examples/marbot/main.tf [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+
+module "alarms" {
+  source         = "github::https://github.com/bitflight-public/terraform-aws-rds-alerts.git?ref=master"
+  db_instance_id = "${aws_efs_file_system.default.id}"
+}
+
+provider "aws" {
+  region = "${var.region}"
+
+  # Make it faster by skipping something
```
Comment (aknysh) [inline examples/marbot/main.tf:28]: Make it faster by skipping some checks and validations

File: examples/simple/main.tf [hunk 10]
```diff
@@ -0,0 +1,32 @@
+### For connecting and provisioning
+variable "region" {
+  default = "us-east-1"
+}
+
+provider "aws" {
+  region = "${var.region}"
+
+  # Make it faster by skipping something
```
Comment (aknysh) [inline examples/simple/main.tf:9]: Make it faster by skipping some checks and validations

File: examples/simple/main.tf [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+  skip_get_ec2_platforms      = true
+  skip_metadata_api_check     = true
+  skip_region_validation      = true
+  skip_credentials_validation = true
+  skip_requesting_account_id  = true
+}
+
+resource "aws_efs_file_system" "default" {
```
Comment (aknysh) [inline examples/simple/main.tf:17]: why EFS alarms in the `lambda` module?

File: main.tf [hunk 12]
```diff
@@ -0,0 +1,60 @@
+data "aws_caller_identity" "default" {}
+
+# Make a topic
+resource "aws_sns_topic" "default" {
+  name_prefix = "${local.alert_for}-threshold-alerts"
```
Comment (aknysh) [inline main.tf:5]: use `label` with attributes

**Review by aknysh — COMMENTED** [review]
inline comments

**Comment by jamengual** [comment]
@Jamie-BitFlight is this still relevant?

**Comment by jamengual** [comment]
please reopen if needed

**Pull request closed without merging**